gen2-bounding-box-filtering/kalman/README.md

```diff
@@ -0,0 +1,6 @@
+# Kalman filter
```

Comment:
Maybe expand this a tiny bit more - like why is it useful, and perhaps some links to relevant kalman filter tutorials (eg. YT)